e2e-tests/ukef/cypress/e2e/journeys/utilisation-reports/utilisation-report-submission.spec.js

```diff
@@ -0,0 +1,235 @@
+import {
+  UTILISATION_REPORT_RECONCILIATION_STATUS,
+  UtilisationReportEntityMockBuilder,
+  getPreviousReportPeriodForBankScheduleByMonth,
+  toIsoMonthStamp,
+} from '@ukef/dtfs2-common';
+
+import relative from '../../relativeURL';
+import { NODE_TASKS, PDC_RECONCILE, TFM_URL } from '../../../../../e2e-fixtures';
+
+import portalPages from '../../../../../portal/cypress/e2e/pages';
+import tfmPages from '../../../../../tfm/cypress/e2e/pages';
+
+import MOCK_USERS from '../../../../../e2e-fixtures/portal-users.fixture';
+import { tfmFacilityForReport } from '../../../fixtures/tfm-facility';
+import { february2023ExpectedValues } from '../../../fixtures/february-2023-expected-values';
+
+const { BANK1_PAYMENT_REPORT_OFFICER1 } = MOCK_USERS;
+
+context('Portal to TFM utilisation report submission', () => {
```

Comment:
Nice test, I'll add a ticket to do an end to end one up to keying sheets where we create the csv file to test it :)